src/main/java/uk/gov/hmcts/darts/automation/utils/DateUtils.java

```diff
@@ -135,6 +137,20 @@ public static String zonedTimestamp() {
 				.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
 				.format(ZonedDateTime.now());
 	}
+
+	public static String convertToUtc(String dateTime) {
```

Comment:
Have you tested this new method (written unit tests for example) using test cases which are in BST (British Summer Time) as well as winter and confirming the resultant UTC dates are as expected?

Reply:
I have tested different inputs in a separate test project (both during and outside of BST) and results were as expected. I will check if there's a way for me to make this test visible to others as well.